lib/mimic/cover.ex

```diff
@@ -11,6 +11,18 @@ defmodule Mimic.Cover do
     :cover.is_compiled(module) != false
   end
 
+  @doc false
+  def setup_if_enabled do
+    if cover_enabled?(), do: export_private_functions()
+    :ok
+  end
+
+  @doc false
+  def cover_enabled? do
+    # Check to to see if a public function is :cover is started
+    function_exported?(:cover, :start, 0)
```

Comment:
I'm not entirely sure if this will check if cover is not started. This is checking if the module cover is loaded in memory which might be true even without `--cover` I think? Did you check if the start up is faster or if the export_private_functions is actually not called when `--cover` is not passed?

Reply:
Yup I did check that it works. Interestingly the :cover module exists but the check returns false until you call something on the module then the check returns true

Reply:
Yeah this is the default behaviour during development. It only loads a module if referenced. So if any code mentions :cover before Mimic starts it might be loaded and function_exported? will return true

Reply:
I think that's the embedded versus interactive options? (To load the modules dynamically)

Reply:
oops see comment in top level of PR. deleting this